sample/kotlin/src/main/kotlin/com/datadog/android/sample/SampleApplication.kt

```diff
@@ -295,6 +298,7 @@ class SampleApplication : Application() {
             .trackUserInteractions()
             .trackLongTasks(250L)
             .trackNonFatalAnrs(true)
+            .setSlowFrameListenerConfiguration(SlowFrameListenerConfiguration())
```

Comment:
minor: maybe it is worth to have something like `SlowFrameListenerConfiguration.DEFAULT` for the clarity?

Reply:
yep, good suggestion 👍